spec/models/issue_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Issue do
+
+  describe '.sync_with_github' do
+
+      it "deletes issues that are no longer open", :vcr  do
+        issue = FactoryBot.create(:issue)
+
+        Issue.sync_with_github
+        expect(Issue.all).to be_empty
+      end
+
+      it "updates issues that are open", :vcr do
+        user_bot = FactoryBot.create(:user,
+                                    name: "ombu-bot",
+                                    id: 1)
```

Comment:
In out next pair session we will dedicate the two hours to forget this kind of indentation 😬 


![Screen Shot 2020-05-20 at 8 48 10 AM](https://user-images.githubusercontent.com/4595174/82460675-b31c0d80-9a76-11ea-8ddb-e13c35711177.png)

Reply:
Yes sir!